tests/templates/test_subroutines/test_trotter.py

```diff
@@ -211,6 +213,93 @@
     }
 )
 
+test_resources_data = {
+    (0, 1): Resources(
```

Comment:
A comment here defining `(0, 1)` will be nice.

Reply:
Its consistent with the other tests for trotter (see for example the `TestDecomposition`), I can add a comment

Reply:
Added